src/webview/create-component/pages/fromExistingGitRepo.tsx

```diff
@@ -229,7 +249,27 @@ export function FromExistingGitRepo({ setCurrentView }) {
                                                 spacing={2}
                                                 alignItems="center"
                                             >
-                                                <CircularProgress />
+                                                <Box sx={{ position: 'relative', display: 'inline-flex' }}>
+                                                    <CircularProgress />
```

Comment:
If you use the determinant variant and pass the value then it will show another visual indicator of progress: https://mui.com/material-ui/react-progress/#circular-determinate